catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/testsuites/generateAllEspressoTestsSuite.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+#put header
+echo "/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2017 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */\n" > AllEspressoTestsSuite.java
+
+#put packagename
+echo "package org.catrobat.catroid.uiespresso.testsuites;\n" >> AllEspressoTestsSuite.java
+
+#put imports
+find .. -name '*Test.java' | sed 's/\.\.\//import org.catrobat.catroid.uiespresso./' | sed 's/\//\./g' | sed 's/.java/;/' | sort >> AllEspressoTestsSuite.java
+
+#put annotations
+echo "import org.junit.runner.RunWith;\nimport org.junit.runners.Suite;\n\n@RunWith(Suite.class)\n@Suite.SuiteClasses({" >> AllEspressoTestsSuite.java
+
+#put classes to be imported
+find .. -name '*Test.java' | sed 's/^.*\///' | sed 's/java/class/' | sed 's/$/,/' | sed '$s/,//' | sed 's/^/		/' >> AllEspressoTestsSuite.java
+
+#put class
+echo "})\npublic class AllEspressoTestsSuite {\n}" >> AllEspressoTestsSuite.java
+
+exit
```

Comment:
Maybe this file should be executable (`chmod +x generateAllEspressoTestsSuite.sh`)? So everyone can easily execute it.
However, echo in bash doesn't convert `\n` into new line characters by default. You either have to use `#!/bin/sh`, `echo -e` or `printf`. Or if you like inlines or variables instead of comments, take a look at [these alternatives](https://gist.github.com/robertpainsi/4b4267bc8ff96dc5c689d058f8e32149).

Interesting to see that your SuiteClasses (classes to be imported) is completely different from mine :laughing: